tbtc-ts/src/bitcoin.ts

```diff
@@ -110,13 +184,99 @@ export interface Client {
    */
   getRawTransaction(transactionHash: string): Promise<RawTransaction>
 
+  /**
+   * Gets the number of confirmations that a given transaction has accumulated
+   * so far.
+   * @param transactionHash - Hash of the transaction.
+   * @returns The number of confirmations.
+   */
+  getTransactionConfirmations(transactionHash: string): Promise<number>
+
+  /**
+   * Gets height of the latest mined block.
+   * @return Height of the last mined block.
+   */
+  latestBlockHeight(): Promise<number>
+
+  /**
+   * Gets concatenated chunk of block headers built on a starting block.
+   * @param blockHeight - Starting block height.
+   * @param confirmations -  Number of confirmations (subsequent blocks) built
```

Comment:
Maybe `chainLength`? I think `confirmations` are not valid in this context.

Reply:
Done.